pkgs/development/interpreters/supercollider/default.nix

```diff
@@ -1,6 +1,8 @@
 { stdenv, fetchurl, cmake, pkgconfig
 , libjack2, libsndfile, fftw, curl, gcc
 , libXt, qtbase, qttools, qtwebkit, readline
+, withAlsa ? !stdenv.isDarwin, alsaLib             # ALSA is required for MIDI to work in SC
+, withAvahi ? false, avahi                         # Avahai is optional dependency
```

Comment:
`qtbase` already depends on `avahi`, no need to make it optional here.

Reply:
the cmake build system doesn't find avahi unless specifically added to dependencies

Reply:
Yes, please add it as a dependency, but remove `withAvahi` option.